packages/loot-design/src/components/common.js

```diff
@@ -111,18 +111,27 @@ export function AnchorLink({
   );
 }
 
-export const ExternalLink = React.forwardRef((props, ref) => {
-  function onClick(e) {
-    e.preventDefault();
-    window.Actual.openURLInBrowser(props.href);
-  }
+export const ExternalLink = React.forwardRef(
+  ({ asAnchor, children, ...props }, ref) => {
+    function onClick(e) {
+      e.preventDefault();
+      window.Actual.openURLInBrowser(props.href);
+    }
 
-  if (props.asAnchor) {
-    // eslint-disable-next-line
-    return <a ref={ref} {...props} onClick={onClick} />;
+    if (asAnchor) {
```

Comment:
```
index.js:1451 Warning: React does not recognize the `asAnchor` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `asanchor` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
    in a (at common.js:122)
    in ForwardRef (at WelcomeScreen.js:27)
    in div (at common.js:642)
    in P (at WelcomeScreen.js:25)
```

Reply:
Functionally this component has really no changes. It just looks like a lot due to prettier re-formating things.